specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2024-10-01-preview/examples/ApiManagementGatewayGetKubernetesManagedGateway.json

```
@@ -0,0 +1,52 @@
+{
+  "parameters": {
+    "gatewayName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2024-10-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.ApiManagement/gateway/example-gateway",
+        "name": "example-gateway",
+        "type": "Microsoft.ApiManagement/gateway",
+        "kind": "KubernetesManaged",
+        "tags": {
+          "owner": "v-aswmoh",
+          "ReleaseName": "Z3"
+        },
+        "location": "East US",
+        "etag": "AAAAAAAWN/4=",
+        "properties": {
+          "targetProvisioningState": "",
+          "createdAtUtc": "2021-06-16T09:40:00.9453556Z",
+          "frontend": {
+            "defaultHostname": "example-gateway.eastus.gateway.azure-api.net"
+          },
+          "backend": {
+            "subnet": {
+              "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.Network/virtualNetworks/vn1/subnets/sn1"
+            }
+          },
+          "configurationApi": {
+            "hostname": "tenant1.configuration.eastus-01.azure-api.net"
+          }
```

This is deprecated and won't be in new gateway SKU anymore:
```suggestion
          }
```

@solankisamir I guess we have to keep this in forever and can never remove it? What if this was never officially shipped as public preview nor GA? (don't think this changes anything, but one can hope)

---

Removed it, let me know if we need to keep it for backwards compat.